tensorrt_llm/llmapi/llm_args.py

```diff
@@ -822,24 +813,13 @@ class LlmArgs(BaseModel):
         default=None,
         description="The maximum number of iterations for request stats.")
 
-    workspace: Optional[str] = Field(default=None,
-                                     description="The workspace for the model.")
-
     # A handful of options from PretrainedConfig
     embedding_parallel_mode: str = Field(
         default='SHARDING_ALONG_VOCAB',
         description="The embedding parallel mode.")
 
     fast_build: bool = Field(default=False, description="Enable fast build.")
```

Comment:
Please move `fast_build` to `TrtLlmArgs`

Reply:
Done, thanks.